client/signup/steps/goals/types.ts

```diff
@@ -0,0 +1,7 @@
+import { GoalId } from './constants';
+
+export type Goal = {
+	id: GoalId;
+	label: React.ReactChild | string;
```

Comment:
Is `React.ReactChild` a possible returned type of `useTranslate`?

Reply:
Not of the `useTranslate` but the `translate` function which is returned by `useTranslate`.

Reply:
Gotcha.